funsor/__init__.py

```diff
@@ -24,14 +24,15 @@
     memoize,
     minipyro,
     montecarlo,
-    numpy,
     ops,
     sum_product,
     terms,
     testing,
-    torch
 )
 
+set_backend(get_backend())
```

Comment:
It seems a little safer to execute this in `funsor.util` where `set_backend()` is defined. That way anything depending on `get_backend()` will execute after this statement. Do you have a reason for locating it here in `__init__.py`?

Reply:
There is no reason besides let the backend executed by default. I'll move it there.